### PR TITLE
Bug 2019198:Rename Prometheus metrics

### PR DESCRIPTION
**Pull request by aneeshkp**
Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>
Address the Bug, fixes renaming of offset metrics. Once the PR is merged, need to fix the alert rule in ptp-operator

This PR  fixes the renaming of metrics as per the source of the offset is reported in the log. As of now, we have three sources phc, system, and master offset. The metrics are reported as follows
Example logs
``` 
phc2sys[345276.663]: `master` offset       -265 s2 freq   -2265 path delay        87
phc2sys[345276.663]: [ptp4l.0.config] CLOCK_REALTIME `phc` offset         9 s2 freq  -78510 delay   1108
phc2sys[345276.663]: [ptp4l.0.config] ens5f0 `phc` offset         7 s2 freq   -2248 delay   2893
phc2sys[345276.663]: [ptp4l.0.config] ens5f2 `sys` offset         77 s2 freq   -2248 delay   2893

````

Renames `openshift_ptp_offset_from_master`  to 
` openshift_ptp_offset_ns ` and adds label  called `from` with values `phc|sys|master`
``` 
openshift_ptp_offset_ns{from ="master" iface="ens5fx",node="cnfde7.ptp.lab.eng.bos.redhat.com",process="ptp4l"}
openshift_ptp_offset_ns{from="phc" iface="CLOCK_REALTIME",node="cnfde7.ptp.lab.eng.bos.redhat.com",process="phc2sys"}
openshift_ptp_offset_ns{from="phc" iface="ens5f1",node="cnfde7.ptp.lab.eng.bos.redhat.com",process="phc2sys"}
openshift_ptp_offset_ns{from="sys" iface="ens5f2",node="cnfde7.ptp.lab.eng.bos.redhat.com",process="phc2sys"}

openshift_ptp_delay_ns{from="master", iface="ens5fx",node="cnfde7.ptp.lab.eng.bos.redhat.com",process="ptp4l"} 84
openshift_ptp_frequency_adjustment_ns{from="master", iface="ens5fx",node="cnfde7.ptp.lab.eng.bos.redhat.com",process="ptp4l"} -2197
```

### CODE REVIEW

**Comment by SchSeba** [comment]
/cc @josephdrichard 

please help me reviewing this PR

**Review by josephdrichard — COMMENTED** [review]
Can you add a commit message to indicate what changes you are making here?

**Review by SchSeba — COMMENTED** [review]
Also will this changes work with both print modes?

summary_interval= 1
summary_interval= -4

File: pkg/daemon/metrics.go [hunk 1]
```diff
@@ -129,23 +101,19 @@ var (
 		prometheus.GaugeOpts{
 			Namespace: PTPNamespace,
 			Subsystem: PTPSubsystem,
-			Name:      "ptp_interface_role",
-			Help:      "0 = PASSIVE 1 = SLAVE 2 = MASTER 3 = FAULTY",
+			Name:      "interface_role",
+			Help:      "0 = PASSIVE, 1 = SLAVE, 2 = MASTER, 3 = FAULTY,  4 = UNKNOWN",
```
[the remark this answers is not in the document]
Comment (aneeshkp) [reply pkg/daemon/metrics.go:105]: Yes it works with both , if there is no offset in the logs , then there will be no metrics

**Comment by aneeshkp** [comment]
> 

I have described the PR in detail in the description section. That should suffice

File: pkg/daemon/metrics.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (aneeshkp) [reply pkg/daemon/metrics.go]: removed the double space

**Comment by SchSeba** [comment]
@aneeshkp please squash the commits into one.

also please add like one or two sentences about this change in the commit message so it will be easy to find and revert if needed

**Comment by aneeshkp** [comment]
updated commit message

**Pull request merged**